slither/tools/read_storage/read_storage.py

```diff
@@ -119,7 +127,32 @@ def get_storage_layout(self) -> None:
                 elif isinstance(type_, ArrayType):
                     elems = self._all_array_slots(var, contract, type_, info.slot)
                     tmp[var.name].elems = elems
-
+        for contract, var in self.constant_slots:
+            var_name = var.name
```

Comment:
Can you not get the literal expression without this?

Reply:
Which line(s) are you referring to?